package-lock.json

```diff
@@ -1,84 +1,57 @@
 {
   "name": "optic-release-automation-action",
   "version": "2.3.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Could you add the `.nvmrc` file?
We are running node12 here and the previous lock file was wrong. These changes are good!